cfme/containers/image.py

```diff
@@ -123,9 +125,19 @@ def compliant(self):
             raise ValueError("{} is not a known state for compliance".format(text))
 
 
+class ImageAllView(BaseLoggedInPage):
+
+    table = Table(locator="//div[@id='list_grid']//table")
+
+    @property
+    def is_displayed(self):
+        return match_page(summary='Container Images')
+
+
 @navigator.register(Image, 'All')
 class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+    VIEW = ImageAllView
 
     def am_i_here(self):
```

Comment:
Remove am_i_here method as you have in `ContainerAll` class, its covered by VIEW.is_displayed.